site/README.md

```diff
@@ -65,7 +65,7 @@ The non-versioned site pages are all the `/site/docs/.*md` files and the docs ar
 > This build process is currently missing older versions and the javadoc branches.
 > Until these branches are merged, these steps will not work.
 
-All previously versioned docs will be committed in `docs-<version>` branches and mounted using [git worktree](https://git-scm.com/docs/git-worktree) at build time. The worktree will pull these versions in following the `/site/docs/docs/<version>` convention. The `latest` version, will be a secondary copy of the most recent build version in the worktree, but pointing to `/site/docs/docs/latest`. There is also a `javadoc` branch that contains all prior static generation versions of the javadocs in a single tag.
+The Iceberg versioned docs are committed in the `docs` branch and mounted using [git worktree](https://git-scm.com/docs/git-worktree) at build time. The worktree pulls these version directories in following the `/site/docs/docs/<version>` convention. The `latest` version, will be a secondary copy of the most recent build version in the worktree, but pointing to `/site/docs/docs/latest`. There is also a `javadoc` branch that contains all prior static generation versions of the javadocs.
```

Comment:
Are all docs in the same branch?

Reply:
Yes, I moved to this assumption so we don't have to make branches per version anymore. I think that will make changes across multiple docs much faster.